sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/organisms/SummaryRowViewScreen.kt

```diff
@@ -0,0 +1,273 @@
+package uk.gov.hmrc.sample_compose_fragments.presentation.screens.organisms
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import uk.gov.hmrc.components.compose.molecule.multiColumnRowView.MultiColumnRowItem
+import uk.gov.hmrc.components.compose.molecule.multiColumnRowView.MultiColumnRowView
+import uk.gov.hmrc.components.compose.organism.HmrcCardView
+import uk.gov.hmrc.components.compose.organism.container.SeparatedViewContainer
+import uk.gov.hmrc.components.compose.organism.summaryrow.SummaryRowView
+import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme.dimensions
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ExamplesSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.PlaceholderSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ScreenScrollViewColumn
+
+object SummaryRowViewScreen {
+
+    @Composable
+    operator fun invoke(onClickAction: () -> Unit) {
```

Comment:
Is there any advantage to this being an object, rather than just a top level composable function?
```suggestion
@Composable
fun SummaryRowViewScreen(onClickAction: () -> Unit) {
```